### PR TITLE
docs(Ref): add a basic example

### PR DESCRIPTION
**Pull request by layershifter**
This PR adds a basic example to the `Ref` component.

### CODE REVIEW

File: docs/app/Examples/modules/Popup/Types/PopupExampleTitled.js [hunk 1]
```diff
@@ -18,17 +18,13 @@ const users = [
...
 ]
-/* TODO: Remove <div> wrapper after all our components will be classes */
+
 const PopupExampleTitled = () => (
   <div>
     {users.map(user => (
       <Popup
         key={user.name}
-        trigger={(
-          <div>
```
Comment (layershifter) [inline docs/app/Examples/modules/Popup/Types/PopupExampleTitled.js:28]: After #2219 was merged stateless component can be used as `trigger` without warnings :+1:

**Comment by layershifter** [comment]
@levithomason Feel free to push any updates there if you will want :smile:

**Comment by levithomason** [comment]
Awesome 👍

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.76.0`

**Pull request merged**